molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -20,7 +20,7 @@
 
     - name: Create Dockerfiles from image names
       template:
-        src: "{{ molecule_scenario_directory }}/Dockerfile.j2"
+        src: "{{ molecule_scenario_directory }}/{{ molecule_yml.provisioner.dockerfile | default('Dockerfile.j2') }}"
```

Comment:
We should retain the default behavior here.

```
src: molecule_yml.provisioner.dockerfile | default("{{ molecule_scenario_directory }}/Dockerfile.j2")
```

Reply:
I actually had it that way first. The issue with that is the  search path is under `.../molecule/provisioner/ansible/playbooks/docker/` (relative to the default create.yml) so if you just specify `Dockerfile2.j2` for example it wouldn't work. The way I have it, it is relative to the scenario path so you can have `Dockerfile2.j2` in your scenario directory and just refer to it that way.

The default is already there to preserve the existing default behavior.